value-processor/src/org/immutables/value/processor/Immutables.generator

```diff
@@ -1872,7 +1872,7 @@ checkNotIsSet([v.names.isSet](), "[v.names.raw]");
       [else]
     this.[v.name].put(
         [requireNonNull type](key, "[v.name] key"),
-        [requireNonNull type](value, "[v.name] value"));
+        [requireNonNull type](value, "[v.name] value for key: " + key));
```

Comment:
this one should be changed too, or was this one left out intentionally? thank you

Reply:
see my question above about how to check whether value is a primitive or not

Reply:
yes, reading and thinking, thank you